FIX: Create virtual environment per Python version (#4681)
ansys/pyaedt: doc/source/Getting_started/Installation.rst

  :alt: PyAEDT toolkit buttons available in AEDT

Extension manager
~~~~~~~~~~~~~~~~~
The user can install or uninstall automated workflows using the extension manager.
There are three options:

- Custom PyAEDT scripts.

> - PyAEDT workflows. Existing workflows in the PyAEDT library.

Reviewer: ```suggestion
- Existing workflows in the PyAEDT library.
```

> - Open source PyAEDT toolkits. Open source PyAEDT toolkits following the `common library <https://aedt.common.toolkit.docs.pyansys.com/>`_ standard.

Reviewer: ```suggestion
- Open source PyAEDT toolkits described in the `PyAEDT Common Toolkit <https://aedt.common.toolkit.docs.pyansys.com/>`_ documentation.
```
The title for the referenced doc should be "PyAEDT Comon Toolkit" documentation followed by the version. I opened a PR to fix it.

- Custom PyAEDT scripts.

- Existing workflows in the PyAEDT library.

> - Open source PyAEDT toolkits described in the `PyAEDT Common Toolkit <https://aedt.common.toolkit.docs.pyansys.com/>`_ documentation.

Reviewer: ```suggestion
- Open source PyAEDT toolkits described in the `PyAEDT Common Toolkit documentation <https://aedt.common.toolkit.docs.pyansys.com/>`_.
```
Make "documentation part of the link text since it is where the link goes. (Otherwise, one might think it went to the repository for this library.)
